psshlib/psshutil.py

```diff
@@ -39,15 +40,15 @@ def read_host_file(path, default_user=None, default_port=None):
         line = line.strip()
         if not line or line.startswith('#'):
             continue
-        host, port, user = parse_host_entry(line, default_user, default_port)
-        if host:
+        host, port, user = parse_host_entry(line, default_user, default_port, host_glob)
+        if host and (not host_glob or fnmatch.fnmatch(host, host_glob)):
             hosts.append((host, port, user))
     return hosts
 
 
 # TODO: deprecate the second host field and standardize on the
 # [user@]host[:port] format.
-def parse_host_entry(line, default_user, default_port):
+def parse_host_entry(line, default_user, default_port, host_glob):
```

Comment:
The new `host_glob` argument is unused, right? If so, we'd better remove it.

Reply:
Fixed.